src/facade/dragonfly_connection.cc

```diff
@@ -733,40 +731,37 @@ void Connection::HandleRequests() {
 
 #ifdef DFLY_USE_SSL
   if (ssl_ctx_) {
-    const bool no_tls_on_admin_port = GetFlag(FLAGS_no_tls_on_admin_port);
-    if (!(IsPrivileged() && no_tls_on_admin_port)) {
-      // Must be done atomically before the premption point in Accept so that at any
```

Comment:
removed `if (!(IsPrivileged() && no_tls_on_admin_port)) {` condition.
everything else is just indentation